### ISSUE TITLE
add new tables: team, user_team

### ISSUE THREAD

**Issue by joemcl** [ProgressiveCoders/Spoke#639]
It would be useful to be able to add users to, and allow a user to join, 'teams' of users.

Teams could then be organized and based on those volunteering for a particular candidate/political campaign, an independent expenditure campaign, an issue campaign, an event, by geography, various demographics, language fluency, etc. 

For example - a Spoke user organization may want to allow and encourage friendly competition between Eastern time zone and Pacific time zone users, or to be able to quickly identify users fluent in Spanish.

**Comment by harpojaeger** [comment]
Would the existing Organizations model not work for this?

**Comment by joemcl** [comment]
@harpojaeger I don't think so. Every user must be a member of an Organization. An Organization should be able to have many teams 'under'/part of it, and a user should be able to be a member of multiple teams that are part of the same Organization. In one Organization, we might want teams for EST texters, CST texters, MST texters and PST texters, so we can create queries/'league tables' to show the total number of texts sent by each time zone team. Users in each of those time zone teams could also be members of a  Spanish speakers/writers texters team to which we might re-assign replies received in Spanish.

**Comment by ibrand** [comment]
Note: There is a Politics Rewired feature called Teams that allows for organizing volunteers. I think this issue would work the same way.

**Comment by joemcl** [comment]
yup @ibrand probably

**Comment by Frydafly** [comment]
@ibrand @ben-pr-p looping you both in since you mentioned politics rewired has this feature?
